app/src/main/java/com/github/studydistractor/sdp/ui/CreateUserScreen.kt

```diff
@@ -163,9 +164,14 @@ fun CreateUserScreen(
         // Validate
         Button( onClick = {
             var isUserCreated = true
-            val birthday = hashMapOf(
-                "year" to year, "month" to month, "day" to dayOfMonth)
-            val user = UserData(userId, firstname, lastname, phone, birthday, 0)
+            val stmp = if (selectedDateText.isNotEmpty()) {
```

Comment:
I think it's better if you do the date validation in the UserViewModel, i.e. just pass the birthday string to UserData and then checkBirthdayFormat from the viewmodel could be something like checkBirthdayFormatAndConvertToTimestamp. That way, the logic of validation will be in the viewmodel where it belongs.